### ISSUE TITLE
Mango Java Script - Services as Utilities

### ISSUE THREAD

**Issue by terrypacker** [infiniteautomation/ma-core-public#1429]
I see more and more people using Java objects directly to access various parts of Mango.  This can be unsafe and dangerous, however the recently implemented Services might be a great way to ensure people don't have to use lower level Java apis.

The service layer has permissions and validation checking and every service class extends `AbstractVOService`.  It would not be very hard to implement some code that scans for all Services and wraps these in a JavaScriptUtility class that applies the script permissions to all calls.  

Basically the Service layer is a much safer way to perform CRUD operations on Mango than a DAO as previously mentioned there are permission and validation checks.  The added benefit would then be that the script would not have to be executed with access to Java objects.

**Comment by terrypacker** [comment]
Not doing this as it is in the new Script environment.
